Add kubearchive test user yaml resources (#456)
kubearchive/kubearchive: DEVELOPMENT.md

    kubectl get -n kubearchive secrets kubearchive-api-server-tls -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
    ```

1. Create a service account with a specific role to test the REST API.

>    You can use the `default` user with `view` privileges provided for the `test` namespace in `test/users/test-user.yaml`

Reviewer: You need a period at the end of the sentence:

```suggestion
   You can use the `default` user with `view` privileges provided for the `test` namespace in `test/users/test-user.yaml`.
```

Author: Fixed

    ```bash
    go run cmd/kubectl-archive/main.go get batch/v1 jobs --token $(kubectl create -n test token default)
    ```

>    **NOTE**: For this to work the `test/users/test-user.yaml` must be applied

Reviewer: You need a period at the end of the sentence:

```suggestion
   **NOTE**: For this to work the `test/users/test-user.yaml` must be applied.
```

Author: Fixed.

    * [Goland instructions](https://golangforall.com/en/post/go-docker-delve-remote-debug.html#goland-ide)
1. Generate traffic:
    * API: Query the API using `curl` or your browser:

>    **NOTE**: For this to work the `test/users/test-user.yaml` must be applied

Reviewer: You need a period at the end of the sentence:

```suggestion
   **NOTE**: For this to work the `test/users/test-user.yaml` must be applied.
```

Author: Fixed.

   export KIND_CLUSTER_NAME=$(kind -q get clusters)
   ```
   **NOTE**: In case you have more than one kind cluster running, manually set the proper one

> 1. Using the `hack/quick-install.sh` script. If you get this error:

Reviewer 2: I think this apply no matter how you install it. I think the more generic version is "operator deployment times does not become ready" or something similar.

Author: I've rephrased it so it still points to an specific step of the development setup but it also specifies that the issue is on deploying the operator.

> 1. Deploying the operator (e.g. using the `hack/quick-install.sh` script), the `Deployment` doesn't reach `Ready` state:

Reviewer: You shouldn't use Latin abbreviations (from Neal) - spell out what you mean instead.

```suggestion
1. Deploying the operator (for example: using the `hack/quick-install.sh` script), the `Deployment` doesn't reach `Ready` state:
```

Author: Done!
